packages/components/src/advertisement/index.stories.jsx

```diff
@@ -0,0 +1,14 @@
+import Advertisement from './index';
+
+export default { title: 'Advertisement', component: Advertisement };
+
+export const Default = () => (
+  <Advertisement
+    id="00000000000e25e9"
+    categories={['Brandenburg/Müllrose/15299/Bart-Müllrose']}
```

Comment:
examples for the other props?

Reply:
not really important as they are not used/solved by adapter and passed into adserver directly. perhaps a link to docu would make sense?

Reply:
yes!